prestoadmin/standalone/config.py

```diff
@@ -140,6 +143,24 @@ def validate_workers(workers):
     return workers
 
 
+def _expand_host(host):
+    m = re.match("(.*)\[(\d{1,})-(\d{1,})\](.*)", host)
```

Comment:
please do not use short names like `m`

please replace to `m` to soemthing like `match` and `num_fnt` to `number_format` (if understand it correctly)